### ISSUE TITLE
feat: Pagination and other components

### ISSUE THREAD

**Issue by KnorpelSenf** [grammyjs/menu#4]
After #3 it should be trivial to implement pagination (both in message and in reply markup) on top of menus. We should consider adding this to the menu plugin.

We could also look into more complex things like a date or a time picker, a calendar entry, radio buttons, check boxes, etc. This would rather go into the direction of `grammy-inline-menu` which is not a goal of this pluign.

Instead, it may make sense to provide a second plugin that has a wealth of useful UI components, and that depends on the menu plugin.

**Comment by felinto-dev** [comment]
Hey!

⚠️ This is NOT a Grammy plugin

I develop an abstraction for pagination in Telegram, you can check here: https://github.com/felinto-dev/tg-menu/blob/master/lib/services/pagination.service.ts

See in action: 

![pagination](https://user-images.githubusercontent.com/32253743/140631707-e004c6a6-bd48-439c-af80-50b524393bdf.gif)

PS: I'm still developing this library and it lacks many things, like documentation and unit tests. However, maybe this code helps in some way so that you don't have to start from scratch.

I'm interested in working on a parallel library that provides the paging buttons for my library and @grammyjs/menu. I believe this can help the community.

**Comment by KnorpelSenf** [comment]
https://github.com/Camelik/menu-pagination
